common/constants.js

```diff
@@ -54,7 +54,8 @@ export const pageIdentifiers = {
   CUSTOMER_UPDATED: 'customer-update',
   DONATION_RECEIVED: 'donation-received',
   DONATION_RECEIPT: 'donation-receipt',
-  PASSWORD_RESET: 'password-reset'
+  PASSWORD_RESET: 'password-reset',
+  GOOGLE_SINGUP: 'google-singup'
```

Comment:
should be signup

Reply:
It is not better a more specific name? tomorrow you could have a local sing-up too.

Reply:
I mean it's spelled signup. A better name though might be password-reset-google, because it's more related to password reset than signup.

Reply:
I Get it, the problem was that the first code was done in another context.
I will fix it.